doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -2725,6 +2726,10 @@ \subsubsection{Surface Inside Face Absorbed Shortwave Radiation Rate {[}W{]}}\la
 
 As of Version 2.1, the previous variable plus absorbed shortwave radiation from uniformly distributed initially-reflected diffuse solar, reflected beam solar, and shortwave radiation from lights. This sum is the power of all sources of solar and visible radiation absorbed by the surface at the inside face.
 
+\subsubsection{Surface Construction Index {[]}}\label{surface-construction-index}
+
+The variable reports an index that points to specific Construction. The variable is an additional output variable when Output:Diagnostics, DisplayAdvancedReportVariables; is given in an input file.
+ 
```

Comment:
@rraustad I don't think it has a direct way to match index and corresponding construction. However, it can be done. Here are steps:
1. Find surface name from Surface Construction Index
2. Search the surface name under HeatTransfer Surface in eio
3. Find the construction name in the 7th item

I am going to provide guidance in IO Ref.